src/controllers/user-controller.js

```diff
@@ -18,6 +19,11 @@ const user = joi.object({
   aliado: joi.string().max(20).required(),
 });
 
+const newPass = joi.object({
+  uuid: joi.string().max(40).required(),
+  password: joi.string().required(),
+});
```

Comment:
```suggestion
const newPass = joi.object({
  uuid: joi.string().guid({ version: 'uuidv4' }).required(),
  password: joi.string().required(),
});
```

also what is the expected validations for the password? length, etc...